mne/channels/channels.py

```diff
@@ -228,6 +230,70 @@ def get_channel_types(self, picks=None, unique=False, only_data_chs=False):
         return _get_channel_types(self.info, picks=picks, unique=unique,
                                   only_data_chs=only_data_chs)
 
+    def get_ch_positions(self, picks=None):
+        """Get a dictionary of channel positions for each channel.
+
+        Channel positions are in meters in the specified coordinate frame
+        of the montage.
+
+        Parameters
+        ----------
+        picks : str | list | slice | None
+            None gets good data indices.
```

Comment:
get ch position for MEG channel is not enough. You need orientation.

```suggestion
            None corresponds to all data channels that are not MEG.
```